test/e2e/storage/testsuites/fsgroupchangepolicy.go

```diff
@@ -0,0 +1,358 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testsuites
+
+import (
+	"strconv"
+
+	"github.com/onsi/ginkgo"
+	v1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	errors "k8s.io/apimachinery/pkg/util/errors"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	e2evolume "k8s.io/kubernetes/test/e2e/framework/volume"
+	"k8s.io/kubernetes/test/e2e/storage/testpatterns"
+	storageutils "k8s.io/kubernetes/test/e2e/storage/utils"
+	utilpointer "k8s.io/utils/pointer"
+)
+
+type fsGroupChangePolicyTestSuite struct {
+	tsInfo TestSuiteInfo
+}
+
+var _ TestSuite = &fsGroupChangePolicyTestSuite{}
+
+// InitFsGroupChangePolicyTestSuite returns fsGroupChangePolicyTestSuite that implements TestSuite interface
+func InitFsGroupChangePolicyTestSuite() TestSuite {
+	return &fsGroupChangePolicyTestSuite{
+		tsInfo: TestSuiteInfo{
+			Name: "fsgroupchangepolicy",
+			TestPatterns: []testpatterns.TestPattern{
+				testpatterns.FsGroupDefaultFsPreprovisionedPV,
+				testpatterns.FsGroupDefaultFsDynamicPVImmediate,
+				testpatterns.FsGroupDefaultFsDynamicPVDelayed,
+				testpatterns.FsGroupFsVolModePreprovisionedPV,
+				testpatterns.FsGroupFsVolModeDynamicPVImmediate,
+				testpatterns.FsGroupFsVolModeDynamicPVDelayed,
+			},
+			SupportedSizeRange: e2evolume.SizeRange{
+				Min: "1Mi",
+			},
+			FeatureTag: "[Feature:FsGroupChangePolicy]",
```

Comment:
We don't need a feature tag since this feature is intree and beta

Reply:
Done